taskchampion/taskchampion/src/server/cloud/gcp.rs

```diff
@@ -244,10 +253,16 @@ mod tests {
         let Ok(bucket) = std::env::var("GCP_TEST_BUCKET") else {
             return None;
         };
+
+        let Ok(credential_path) = std::env::var("GCP_CREDENTIAL_PATH") else {
+            return None;
+        };
+
```

Comment:
Oh, I didn't realize this was in tests. Let's use `GCP_TEST_CREDENTIAL_PATH` here.

Reply:
Okay!